style.css

```diff
@@ -552,6 +581,107 @@ span {
   display: none;
 }
 
+.popup_window,
+.popup_section {
+  display: none;
+}
+
+.popup_window {
+  top: 0;
+  position: fixed;
+  height: 100%;
+  background: #fff;
+  overflow: scroll;
+}
```

Comment:
- [ ] As I mentioned in my previous review to adhere to the project requirement but it seems like you haven't made any changes, please 🙏🏻 🙏🏻 **make your design as close as possible to provide design**, if you look at the below-attached table of design comparison, your design needs some adjustments related to spaces and other small issues. I would request you kindly fix your PopUp issue, and make it looks professional and similar to Figma's design. Below I am going to tell each and everything in more detail in numeric points.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-07-01 22-45-02](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/0122029c-a325-4036-b225-a7d81f281339)

</td>
<td>

![Screenshot from 2023-07-01 22-44-43](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/5be753d1-fe5a-46c5-a14a-99d6a3948d60)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-07-01 22-46-04](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/49b2769b-53cb-49fb-82b5-a9a38a33e679)

</td>
<td>

![Screenshot from 2023-07-01 22-45-39](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/55c1ab77-2f47-4bbb-b53c-35272c3b2c75)

</td>
</tr>
</table>

### 🖊️ Improvements need

1. Your `popUp` close button is not matching with the original design because of an alignment issue. It should be in the right position and please use a good-quality image for your close button.
2. Please look at the skills **boxes** carefully, you need to add spacing around the text and improve your design to match it with the original design.
**3. Kindly make your button look exactly like the *Figma* design because your button background colour is different and your icons are not aligned properly with each other which makes your design looks bad. Kindly look at the original design.**

### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-07-01 22-49-08](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/5d74a7d3-4527-408e-9733-d454eb202a4a)

</td>
<td>

![Screenshot from 2023-07-01 22-47-42](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/fcd052f7-769e-4ce4-a257-b16293088544)

</td>
</tr>
<tr>
<td>

![Screenshot from 2023-07-01 22-49-08](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/5d74a7d3-4527-408e-9733-d454eb202a4a)

</td>
<td>

![Screenshot from 2023-07-01 22-48-55](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/66476329/33532814-5352-4dd2-91ca-0e0ae2ff68d5)

</td>
</tr>
</table>


### 🖊️ Improvements need

#### In your first desktop comparison image you can see that there is unnecessary space between the project description and the buttons.

#### In your second desktop comparison image you can see that your text is even not aligned properly, I mean not centred to the popUp window we look at the original design.